aggregators/converter.go

```diff
@@ -249,6 +271,13 @@ type eventMetricsBuilder struct {
 	keyedServiceTransactionMetricsArray [1]*aggregationpb.KeyedServiceTransactionMetrics
 	keyedServiceTransactionMetricsSlice []*aggregationpb.KeyedServiceTransactionMetrics
 
+	// There can be at most ? service instance transaction metric per event.
```

Comment:
```suggestion
	// There can be at most 1 service instance transaction metric per event.
```

The `partitionedMetricsBuilder` will build `CombinedMetrics` for a single `APMEvent`. Thus, there will only be 1 service instance transaction metric per APMEvent.